sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/README.md

```diff
@@ -46,15 +52,21 @@ AzureKeyCredential credential = new AzureKeyCredential("{ApiKey}");
 QuestionAnsweringClient client = new QuestionAnsweringClient(endpoint, credential);
 ```
 
-#### Create a QuestionAnsweringProjectsClient
+#### Create a QuestionAnsweringAuthoringClient
+
+To use the `QuestionAnsweringAuthoringClient`, make sure you use the right namespace:
+
+```C# Snippet:QuestionAnsweringAuthoringClient_Namespace
+using Azure.AI.Language.QuestionAnswering.Projects;
```

Comment:
```suggestion
using Azure.AI.Language.QuestionAnswering.Authoring;
```

Reply:
Thanks. All the markdown locations with code should be fixed when I'm done. We extract from code which is fixed.

Reply:
Mostly, I pushed this early for @AlexanderSher to help investigate Azure/autorest.csharp#2726.